README.md

```diff
@@ -161,7 +161,11 @@ To build, run the following commands:
     $ cd mllib-dal
     $ ./build.sh
 ```
-
+The target can be built against different spark versions by specifying profile with <Spark-x.x.x>. E.g.
+```
+    $ ./build.sh Spark-3.1.1
```

Comment:
Spark-3.1.1 => spark-3.1.1
I think you misunderstand.  Just need to change spark to Spark in the documents. You still use lowercase for profiles.  Same below.